kclvm/sema/src/resolver/tests.rs

```diff
@@ -259,8 +259,9 @@ fn test_lint() {
     let sess = Arc::new(ParseSession::default());
     let mut program =
         load_program(sess.clone(), &["./src/resolver/test_data/lint.k"], None).unwrap();
-    pre_process_program(&mut program);
-    let mut resolver = Resolver::new(&program, Options::default());
+    let opts = Options::default();
```

Comment:
Would U pls add more unit test cases for the option?

Reply:
add ut for this option